src/PDE/Limiter.cpp

```diff
@@ -279,9 +279,11 @@ VertexBasedTransport_P1(
     if (dof_el > 1)
     {
       std::vector< tk::real > phi(ncomp, 1.0);
+      std::set< std::size_t > var;
+      for (std::size_t c=0; c<ncomp; ++c) var.insert(c);
       // limit conserved quantities
       VertexBasedLimiting(U, esup, inpoel, coord, e, rdof, dof_el,
-        ncomp, phi, {0, ncomp-1});
```

Comment:
What is the intention here? To limit only components 0 and ncomp-1 or to limit _all_ scalars between 0 and ncomp-1 inclusive? (It is the first that happens in `VertexBasedLimiting()`, in which also `VarList` does not need to be a set, since there is no search in it.